exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/ParquetGroupScanStatistics.java

```diff
@@ -87,19 +91,14 @@ public void collect(List<RowGroupInfo> rowGroupInfos, ParquetTableMetadataBase p
       long rowCount = rowGroup.getRowCount();
       for (ColumnMetadata column : rowGroup.getColumns()) {
         SchemaPath schemaPath = SchemaPath.getCompoundPath(column.getName());
-        Long previousCount = columnValueCounts.get(schemaPath);
-        if (previousCount != null) {
-          if (previousCount != GroupScan.NO_COLUMN_STATS && column.isNumNullsSet()) {
-            Long newCount = rowCount - column.getNulls();
-            columnValueCounts.put(schemaPath, columnValueCounts.get(schemaPath) + newCount);
-          }
+        MutableLong emptyCount = new MutableLong();
+        MutableLong previousCount =
+            Optional.ofNullable(columnValueCounts.putIfAbsent(schemaPath, emptyCount))
```

Comment:
Usage of `Optional` is not justified here (a fancy way to avoid explicit comparison to `null` that leads to new object allocation).

Reply:
Thanks, replaced with `if (previousCount == null)`